packages/cw2/controllers/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
```

Comment:
What happened to this file? Why `packages/cw2/controllers/`?

Reply:
That's some dark magic my IDE probably did. Good spot, let me double check if there are more such surprises